nest-forms-backend/src/forms/forms.service.ts

```diff
@@ -237,32 +227,31 @@ export default class FormsService {
               ],
     }
 
-    if (ico !== null) {
+    const ico = getUserIco(user)
+    if (ico != null) {
       where.userExternalId = undefined
       where.ico = ico
     }
 
-    const data = userExternalId
```

Comment:
This cannot be invoked without `userExternalId` (it's AuthUser), therefore I removed it.